src/CodeGenHelpers/ClassBuilder.cs

```diff
@@ -50,6 +51,19 @@ internal ClassBuilder(string className, CodeBuilder codeBuilder, bool partial =
 
         public bool IsSealed { get; private set; }
 
+        public ClassBuilder WithSummary(string summary)
+        {
+            _xmlDoc.Summary = summary;
+            return this;
+        }
+
+        public ClassBuilder WithInheritDoc(bool inherit = true, string from = null)
```

Comment:
As I'm looking at this API, I cannot help but think this has the wrong method signature. Can you give me an example where it would make sense to say `WithInheritDoc(false)`?

Reply:
I was following the same "style" as the `Override` method in [MethodBuilder](https://github.com/dansiegel/CodeGenHelpers/blob/master/src/CodeGenHelpers/MethodBuilder.cs#L97). So basically `WithInheritDoc(false)` would remove an existing inheritdoc.

Reply:
Should I change it? If so, do you have any suggestions?

Reply:
I would think that you're more likely to provide a value for from than for inherit... why don't you split this into 2 methods, one that takes from and one that takes the bool to disable the inherit doc. Let's also go ahead and set Inherit to false if someone provides a summary